apps/ogrinfo_lib.cpp

```diff
@@ -2305,18 +2305,18 @@ static std::unique_ptr<GDALArgumentParser> GDALVectorInfoOptionsGetParser(
     argParser->add_argument("-al")
         .store_into(psOptions->bAllLayers)
         .help(_("List all layers (used instead of having to give layer names "
-                "as arguments)"));
+                "as arguments)."));
 
     {
         auto &group = argParser->add_mutually_exclusive_group();
         group.add_argument("-so", "-summary")
             .store_into(psOptions->bSummaryParser)
-            .help(_("Summary only: list all layers (used instead of having to "
-                    "give layer names as arguments)"));
+            .help(_("Summary only: Show only summary information like "
```

Comment:
```suggestion
            .help(_("Summary only: show only summary information like "
```

Reply:
@elpaso , did I accept your suggestion correctly? I am not so familiar with editing in GitHub.

Reply:
You did :+1: